From 6fed34d94d337d1a25c375f819fb8a15129f36a2 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Tue, 12 Sep 2017 13:43:55 +0300
Subject: [PATCH 057/706] enetc: Update driver version to v0.3

This is the first driver version supporting m0164 simulator model.
A few additional features were added since v0.2, but the main purpose
of this version is to port the ENETC driver form model m0163 to
m0164.

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 1dd1283c5e6151e58eeec1ab235100751f44e65f)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc.c b/drivers/net/ethernet/freescale/enetc/enetc.c
index b40fb461571e..946170a82d60 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc.c
@@ -3,7 +3,7 @@
 #include "enetc.h"
 
 #define ENETC_DRV_VER_MAJ 0
-#define ENETC_DRV_VER_MIN 2
+#define ENETC_DRV_VER_MIN 3
 
 #define ENETC_DRV_VER_STR __stringify(ENETC_DRV_VER_MAJ) "." \
 			  __stringify(ENETC_DRV_VER_MIN)
-- 
2.17.1

